Connect4.py

```diff
@@ -1,112 +1,120 @@
 GRID_WIDTH = 7
 GRID_HEIGHT = 6
+GRID_TOP_INDEX = 0
+FIRST_COL_INDEX = 0
+INITIAL_DIAGONAL_INDEX = 3
 
-grid = [["_" for i in range(GRID_WIDTH)] for j in range(GRID_HEIGHT)]
+class Board:
 
-def who(turn):
-    if turn % 2 == 0:
-        return 2
-    else: 
-        return 1
+    def __init__(self):
+        self.grid = [["_" for i in range(GRID_WIDTH)] for j in range(GRID_HEIGHT)]
 
-def playerLetter(player):
-    return "X" if player == 1 else "O"
+    def draw(self):
+        columns = "    ".join([str(i) for i in range(GRID_WIDTH)])
+        print("\n ", columns)
+        for row in self.grid:
+            print(row)
 
-def draw():
-    print("\n  0    1    2    3    4    5    6")
-    for row in grid:
-        print(row)
+    def drop(self, col, player):
+        for i in range(GRID_HEIGHT - 1, -1, -1):
```

Comment:
Would you prefer it if the constants were more explicit, so it is more understandable.
Something like "GRID_HEIGHT - 1" could be "GRID_BOTTOM_INDEX = GRID_HEIGHT - 1". 
This way everything would still be dependent on GRID_HEIGHT or GRID_WIDTH

Reply:
yes